Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 buildMvn {
   publishModDescriptor = 'yes'
   mvnDeploy = 'yes'
-  buildNode = 'jenkins-agent-java11'
+  buildNode = 'jenkins-agent-java17'
```

Comment:
here health check command should be updated to use wget instead of curl https://wiki.folio.org/display/TC/JDK+17+and+Java+17

Reply:
got it, thanks